<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContactDialog</class>
 <widget class="QDialog" name="AddContactDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a contact</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fingerprint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_fingerprint"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_email"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>AddContactDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>AddContactDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_fingerprint</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddContactDialog</receiver>
   <slot>fingerprint_edited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edit_name</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddContactDialog</receiver>
   <slot>name_edited()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>name_edited()</slot>
  <slot>fingerprint_edited()</slot>
 </slots>
</ui>
